src/fsm/main.c

```diff
@@ -312,6 +314,32 @@ xopen(const char *s)
 	return f;
 }
 
+#if 0
+static unsigned int word_maxlen = 16;
+static unsigned int num_words = 8;
+static uint64_t seed = 0xdfa7231bc;
+
+static void
+gen_words(FILE *f, const struct fsm *fsm)
+{
+	struct dfa_wordgen_params params = {
+		.minlen = 1,
+		.maxlen = word_maxlen,
+
+		.prob_stop  = 0.3f,
+		.eps_weight = 0.0f,
+	};
+
+	struct prng_state prng;
+
+	memset(&prng, 0, sizeof prng);
+	prng_seed(&prng, seed);
+
+	fsm_generate_words_to_file(fsm, &params, &prng, num_words, f);
+}
+#endif /* 0 */
+
+
```

Comment:
What's going on here? Is `fsm_generate_words_to_file()` just not present yet?

Reply:
Oh, whoops!  That was part of my on-hold project to generate text directly from a DFA.  Must have slipped in there...